### PR TITLE
Users can add new events & see events V1

### PR DESCRIPTION
**Pull request by Tinusw**
**Placeholder Table for Events/show** 
- Stolen and modified from Andres Admin page
image

**Created Events/New**
image
- Would like to replace f.datetime_select with something nicer like a jquery datetime-picker

**Validations added to Event Model**
- Validates all input
- As defined by the trello card, user can enter either a phone number, or an email address, or both. Validations will check that at least 1 of the 2 are entered.

### CODE REVIEW

File: app/views/events/new.html.haml [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+        = form_for(@event) do |f|
+          - if @event.errors.any?
+            .errors
+              %h2
+                This event could not be saved
+              %h2
+                = pluralize(@event.errors.count, "error")
+              %ul
+                - @event.errors.full_messages.each do |msg|
+                  %li=msg
```
Comment (andres3months) [inline app/views/events/new.html.haml]: We should create a partial for errors that we can reuse

**Comment by andres3months** [comment]
We can have a talk later about using a nicer datepicker plugin, so far the html5 datepicker should suffice

File: app/views/events/new.html.haml (hunk 1 above)
Comment (Tinusw) [inline app/views/events/new.html.haml]: Please let me know if you're happy with the partial.

**Comment by josecoelho** [comment]
👍 looks all right for me.

Nice job!

**Pull request merged**